### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by nicflores**
This addresses issue #10.

### CODE REVIEW

File: src/main/g8/src/test/scala/$package$/AppTest.scala [hunk 1]
```diff
@@ -5,6 +5,8 @@ import org.scalajs.dom.document
 
 import org.scalatest.FunSuite
 
+import $package$.App
```
Comment (shadaj) [inline src/main/g8/src/test/scala/$package$/AppTest.scala:8]: This shouldn't be necessary, since we're already in `$package$`.

Comment (nicflores) [reply src/main/g8/src/test/scala/$package$/AppTest.scala:8]: Hi @shadaj. I know, I thought it was weird that I needed it but I couldn't get it to compile without it. Maybe something else is going on?

Comment (nicflores) [reply src/main/g8/src/test/scala/$package$/AppTest.scala]: Never mind, I just removed it compiled locally. So I'll push the change.

**Comment by nicflores** [comment]
Oops, didn't notice your merge from master, sorry.

**Pull request merged**